lib/browser/navigation.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var Url = require('url');
+
+var _ = require('lodash');
+var debug = require('debug')('testium-driver-sync:navigation');
+
+var makeUrlRegExp = require('./makeUrlRegExp');
+var waitFor = require('./wait');
+
+exports._forwarded = [
+  'refresh',
+  'getUrl',
```

Comment:
The additional comma is actually required by "our" (read: airbnb's) liniting rules. They make a good point as well - there's no good reason not to do it unless the code is supposed to run in IE8.